x-pack/plugins/siem/public/components/charts/areachart.tsx

```diff
@@ -78,8 +82,9 @@ export const AreaChartBaseComponent = React.memo<{
               key={seriesKey}
               name={series.key.replace('Histogram', '')}
               data={series.value}
-              xScaleType={ScaleType.Ordinal}
-              yScaleType={ScaleType.Linear}
+              xScaleType={getOr(ScaleType.Linear, 'configs.series.xScaleType', chartConfigs)}
+              yScaleType={getOr(ScaleType.Linear, 'configs.series.yScaleType', chartConfigs)}
+              timeZone={browserTimezone}
```

Comment:
hey @markov00 , would like to know is that ok to pass `timeZone` anyway regardless of the `scaleType`, or it should only passed if the scaleType is `time`?

Reply:
Turns out it doesn't matter if the scaleType is not `Time`, so leave it here to reduce the configs that have to be passed in.